src/main/java/duke/component/StorageStub.java

```diff
@@ -0,0 +1,18 @@
+package duke.component;
+
+import duke.task.Task;
+
+public class StorageStub implements Storage {
+    @Override
+    public TaskList getList() {
+        return new TaskList();
+    }
+
+    @Override
+    public void addToList(Task task) {
+    }
+
+    @Override
+    public void reWrite(TaskList list) {
```

Comment:
since rewrite is a proper word, perhaps you could just name this method as `rewrite`?